tests/api2/test_420_smb.py

```diff
@@ -312,8 +312,8 @@ def check_groupmap_for_entry(groupmaps, nt_name):
 
 
 AUDIT_FIELDS = [
-    'aid', 'vers', 'time', 'addr', 'user', 'sess', 'svc',
-    'svc_data', 'event', 'event_data', 'success'
```

Comment:
I'll approve, but I do note that we used to have `vers` here (and _only_ here?).  Presumably it was originally planned, but got dropped?

Reply:
Yeah, it got removed as part of review during second part of merges.